packages/flutter/test/material/floating_action_button_theme_test.dart

```diff
@@ -315,8 +317,33 @@ void main() {
       'extendedIconLabelSpacing: 12.0',
       'extendedPadding: EdgeInsetsDirectional(7.0, 0.0, 8.0, 0.0)',
       'extendedTextStyle: TextStyle(inherit: true, letterSpacing: 2.0)',
+      'mouseCursor: MaterialStateMouseCursor(clickable)',
     ]);
   });
+
+  testWidgets('FloatingActionButton.mouseCursor uses FloatingActionButtonThemeData.mouseCursor when specified.', (WidgetTester tester) async {
```

Comment:
there are a ton of tests above, like 
- **FloatingActionButtonThemeData values are used when no FloatingActionButton properties are specified**, 
- **FloatingActionButton values take priority over FloatingActionButtonThemeData values when both properties are specified**,
- and for named constructors of `FloatingActionButton`

IMO this should be tested there instead of a separate test. WDYT?